docs/api-aggregation-setup.md

```diff
@@ -0,0 +1,203 @@
+cat >> ${SVCCAT_CA_SETUP} << EOF
+{
+    "hosts": [ ${ALT_NAMES} ],
+    "key": {
+        "algo": "rsa",
+        "size": 2048
+    },
+    "names": [
+        {
+            "C": "US",
+            "L": "san jose",
+            "O": "kube",
+            "OU": "WWW",
+            "ST": "California"
+        }
+    ]
+}
+EOF
+
+
+cfssl genkey --initca ${SVCCAT_CA_SETUP} | cfssljson -bare ${CA_NAME}
+# now the files 'ca.csr  ca-key.pem  ca.pem' exist
+
+export SVCCAT_CA_CERT=${CA_NAME}.pem
+export SVCCAT_CA_KEY=${CA_NAME}-key.pem
+
+export PURPOSE=server
+echo '{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","'${PURPOSE}'"]}}}' > "${PURPOSE}-ca-config.json"
+
+echo '{"CN":"'${SVCCAT_SERVICE_NAME}'","hosts":['${ALT_NAMES}'],"key":{"algo":"rsa","size":2048}}' \
+ | cfssl gencert -ca=${SVCCAT_CA_CERT} -ca-key=${SVCCAT_CA_KEY} -config=server-ca-config.json - \
+ | cfssljson -bare apiserver
+
+export SC_SERVING_CA=${SVCCAT_CA_CERT}
+export SC_SERVING_CERT=apiserver.pem
+export SC_SERVING_KEY=apiserver-key.pem
+```
+
+## Get the appropriate tls ca/cert/key from kube
+
+If you are in a cloud provider environment, you most likely do not
+have access to the appropriate keys.
+
+The key we are looking for in an already running system is the
+`--root-ca-file` flag to the controller-manager.
+
+The following is an example based on a real kubernetes cluster. The
+various files may be named differently and in different locations.
+
+
+```
+export SERVING_NAME=server-ca
+export SERVINGCA_CERT=${SERVING_NAME}.crt
+export SERVINGCA_KEY=${SERVING_NAME}.key
+# a default location
+cp /var/run/kubernetes/${SERVINGCA_CERT} .
+cp /var/run/kubernetes/${SERVINGCA_KEY} .
+```
+
+### Create a cfssl config file for a new signing key
+
+```
+export PURPOSE=server
+echo '{"signing":{"default":{"expiry":"43800h","usages":["signing","key encipherment","'${PURPOSE}'"]}}}' > "${PURPOSE}-ca-config.json"
+```
+
+### Use the existing keys plus the config file to generate the new signing certificate and key
+
+```
+export NAME_SPACE=catalog
+export SERVICE_NAME=catalog-catalog-apiserver
+export ALT_NAMES="\"${SERVICE_NAME}.${NAME_SPACE}\",\"${SERVICE_NAME}.${NAME_SPACE}.svc"\"
+echo '{"CN":"'${SERVICE_NAME}'","hosts":['${ALT_NAMES}'],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -ca=${SERVINGCA_CERT} -ca-key=${SERVINGCA_KEY} -config=server-ca-config.json - | cfssljson -bare apiserver
+```
+
+### Final Key Names
+
+These variables define the final names of the resulting keys.
+
+```
+export SC_SERVING_CA=${SERVINGCA_CERT}
+export SC_SERVING_CERT=apiserver.pem
+export SC_SERVING_KEY=apiserver-key.pem
+```
+
+## Install the Service Catalog Chart with Helm
+
+Use helm to install the Service Catalog into the configured name and
+namespace. This command also enables authentication and aggregation
+and provides the keys we just generated inline.
+
+```
+helm install /root/go/src/github.com/kubernetes-incubator/service-catalog/charts/catalog \
```

Comment:
@arschles any thoughts on using ${GOPATH} as reference? Probably best to still just pretend we're in the project root.